pkg/agent/controller/networkpolicy/status_controller_test.go

```diff
@@ -99,6 +100,25 @@ func TestSyncStatusForNewPolicy(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:          "has invalid rules",
+			policy:        policyWithMultipleRules,
+			realizedRules: 1,
+			invalidRules:  1,
+			expectedStatus: &v1beta2.NetworkPolicyStatus{
+				ObjectMeta: v1.ObjectMeta{
+					Name: policyWithMultipleRules.Name,
+				},
+				Nodes: []v1beta2.NetworkPolicyNodeStatus{
+					{
+						NodeName:           testNode1,
+						Generation:         1,
+						RealizationFailure: true,
+						Message:            `[{"rule_id":"2e8b15c49b690893","reason":"AntreaProxy is not enabled"}]`,
```

Comment:
IIRC, this `Message` will be included in the Status for the network policy CR, using a `NetworkPolicyCondition`.
Should we make some transformation in the controller, given that the rule ID is not meaningful to users?

Reply:
> Should we make some transformation in the controller, given that the rule ID is not meaningful to users?

You are right, the rule ID is not meaningful to users. Do you mean that we should provide more information in the message? Like Service name and namespace? Or something else?